### PR TITLE
fix: fixed the log and file artifact issue

### PR DESCRIPTION
**Pull request by Peeeekay**
This fixes: 

1. user can now see file artifacts 
2. user can switch between logs for each service.

### CODE REVIEW

File: engine/frontend/src/api/container.js [hunk 1]
```diff
@@ -16,30 +16,59 @@ export const runStarlarkPackage = async (url, packageId) => {
...
-            ports: ports,
-        }
-    }) 
+    const servicesPromise = getDataFromApiContainer(makeGetServiceRequest, processServiceRequest)
+    const fileArtifactsPromise = getDataFromApiContainer(makeFileArtifactRequest, processFileArtifactRequest)
 
-    return services;
+    const [services, artifacts] = await Promise.all([servicesPromise, fileArtifactsPromise])
+    return { services, artifacts}
 }
```
Comment (leoporoli) [inline engine/frontend/src/api/container.js]: nit: missing trailing newline

File: engine/frontend/src/api/enclave.js [hunk 2]
```diff
@@ -58,7 +58,6 @@ export const createEnclave = async () => {
 }
 
 export const runStarlark = async(apiClient, packageId) => {
-    console.log(apiClient, packageId)
     const stream = await runStarlarkPackage(apiClient, packageId)
     return stream;
 }
```
Comment (leoporoli) [inline engine/frontend/src/api/enclave.js]: nit: missing trailing newline

File: engine/server/webapp/asset-manifest.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
     "main.7b6afa70.css.map": "/static/css/main.7b6afa70.css.map",
-    "main.caf7863c.js.map": "/static/js/main.caf7863c.js.map"
+    "main.2f3b687c.js.map": "/static/js/main.2f3b687c.js.map"
   },
   "entrypoints": [
     "static/css/main.7b6afa70.css",
-    "static/js/main.caf7863c.js"
+    "static/js/main.2f3b687c.js"
   ]
 }
```
Comment (leoporoli) [inline engine/server/webapp/asset-manifest.json:13]: nit: missing trailing newline

File: engine/server/webapp/index.html [hunk 4]
```diff
@@ -1 +1 @@
-<!doctype html><html lang="en"><head><meta charset="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="theme-color" content="#000000"/><title>Kurtosis Frontend</title><script defer="defer" src="/static/js/main.caf7863c.js"></script><link href="/static/css/main.7b6afa70.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div></body></html>
\ No newline at end of file
+<!doctype html><html lang="en"><head><meta charset="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="theme-color" content="#000000"/><title>Kurtosis Frontend</title><script defer="defer" src="/static/js/main.2f3b687c.js"></script><link href="/static/css/main.7b6afa70.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div></body></html>
```
Comment (leoporoli) [inline engine/server/webapp/index.html:1]: nit: missing trailing newline

**Review by leoporoli — APPROVED** [review]

File: engine/server/webapp/asset-manifest.json (hunk 3 above)
Comment (Peeeekay) [reply engine/server/webapp/asset-manifest.json:13]: this is auto-generated files; so will ignore that.

File: engine/server/webapp/index.html (hunk 4 above)
Comment (Peeeekay) [reply engine/server/webapp/index.html:1]: this is auo-generated file will ignore this.

**Pull request merged**